mowgli_etl/pipeline/wdc/Data_Exploration/offers_corpus_english_v2_parse.py

```diff
@@ -0,0 +1,147 @@
+import json
+import spacy
+import re
+from os import listdir
+
+print("\nLoading NLP library...")
+nlp = spacy.load("en_core_web_sm")
+
+# Generate a list of all possible jsonl files
+directory = [item for item in listdir() if "clean.jsonl" in item]
+if not directory:
+	print("Sorry, there don't seem to be any clean files. Please make sure to run offers_corpus_english_v2_clean.py, or have a file ending in \"clean.jsonl\"")
+	exit(1)
+
+# Present options for jsonl files
+print("Please choose which of the following files you'd like to clean [enter number on the left]:\n")
+for i in range(0, len(directory)):
+	print("\t{} - {}".format(i, directory[i]))
+
+# Have user select which file to clean
+choice = directory[int(input("\nChosen file number: "))]
+print()
+
+data = []
+
+# Read the source data file into a list of JSON data objects
+with open(choice) as file:
+	for line in file:
+		data.append(json.loads(line))
+
+count = 1
+
+# For each product
+for line in data:
```

Comment:
It's not a line any more, it's a product or product_json_obj or similar. Please use accurate variable names.